Bottom: f99078587eb5f60233498f63cf2e9c7929d4b527
Top:    611951f32d8a47112e4b76c3035723317529644f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-15 02:19:53 +0000

Refresh of remove-hid-set_draw_xor-api

---

diff --git a/src/hid/common/hidnogui.c b/src/hid/common/hidnogui.c
index 47bd421..6bc7069 100644
--- a/src/hid/common/hidnogui.c
+++ b/src/hid/common/hidnogui.c
@@ -448,21 +448,6 @@ common_nogui_init (HID *hid)
   hid->invalidate_all =       nogui_invalidate_all;
   hid->set_layer =            nogui_set_layer;
   hid->end_layer =            nogui_end_layer;
-<<<<<<< current
-=======
-  hid->make_gc =              nogui_make_gc;
-  hid->destroy_gc =           nogui_destroy_gc;
-  hid->use_mask =             nogui_use_mask;
-  hid->set_color =            nogui_set_color;
-  hid->set_line_cap =         nogui_set_line_cap;
-  hid->set_line_width =       nogui_set_line_width;
-  hid->set_draw_faded =       nogui_set_draw_faded;
-  hid->draw_line =            nogui_draw_line;
-  hid->draw_arc =             nogui_draw_arc;
-  hid->draw_rect =            nogui_draw_rect;
-  hid->fill_circle =          nogui_fill_circle;
-  hid->fill_polygon =         nogui_fill_polygon;
->>>>>>> patched
   hid->fill_pcb_polygon =     nogui_fill_pcb_polygon;
   hid->fill_pcb_pad =         nogui_fill_pcb_pad;
   hid->thindraw_pcb_pad =     nogui_thindraw_pcb_pad;
